backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/PartialOktaUser.java

```diff
@@ -0,0 +1,16 @@
+package gov.cdc.usds.simplereport.idp.repository;
+
+import com.okta.sdk.resource.user.UserStatus;
+import gov.cdc.usds.simplereport.config.authorization.OrganizationRoleClaims;
+import java.util.Optional;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+public class PartialOktaUser {
+  boolean isAdmin;
```

Comment:
could we add a qualifier for the type of admin this is (org vs site?). As is I'm a bit unclear about what permissions this user has

Reply:
done!